### PR TITLE
two fixes to ePrint IACR

### PR DESCRIPTION
**Pull request by real-or-random**

### CODE REVIEW

File: ePrint IACR.js [hunk 1]
```diff
@@ -56,8 +59,7 @@ function scrape(doc, url) {
 	
 	newItem.date = year;
 	newItem.reportNumber = no;
-	//we want to use this later & make sure we don't make http--> https requests or vice versa. 
-	newItem.url = url.match(/^https?:\/\/[^\/]+/)[0] + "/" + year + "/" + no;
+	newItem.url = "https://eprint.iacr.org/" + year + "/" + no;
```
Comment (adam3smith) [inline ePrint IACR.js]: I still get regular http, e.g. at http://eprint.iacr.org/2005/033 -- please revert this.

File: ePrint IACR.js [hunk 2]
```diff
@@ -36,8 +36,10 @@ function scrape(doc, url) {
 		var no   = reportNo[2];
 	}
 	var title = doc.evaluate(titleXPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+	title = compressSpaces(title);
```
Comment (adam3smith) [inline ePrint IACR.js]: We have `ZU.trimInternal(string)` for this. Don't write a custom function

**Comment by real-or-random** [comment]
Should be better now.

Sorry, I had forgot that I use the HTTPS Everywhere plugin, which redirects automatically.

**Comment by adam3smith** [comment]
thanks!

**Pull request merged**